### PR TITLE
Adding Spark 3.2 and Flink latest version

### PR DESCRIPTION
**Pull request by chethanuk-plutoflume**

### CODE REVIEW

**Comment by eddumelendez** [comment]
Hi @chethanuk-plutoflume, thanks for the PR! We can remove the previous migrations. Can you update the PR, please?

**Comment by eddumelendez** [comment]
@chethanuk-plutoflume ping

**Comment by chethanuk-plutoflume** [comment]
> We can remove the previous migrations

will remove with next version

File: src/main/scala/io/sdkman/changelogs/SparkMigrations.scala [hunk 1]
```diff
@@ -33,4 +33,18 @@ class SparkMigrations {
     ).validate()
       .insert()
   }
+
+  @ChangeSet(
+    order = "020",
+    id = "020-add_spark_3.3.1",
```
Comment (cphbrt) [inline src/main/scala/io/sdkman/changelogs/SparkMigrations.scala:39]: This says `3.3.1` whereas line 45 and 46 refer to `3.3.0`.

**Review by cphbrt — CHANGES_REQUESTED** [review]

**Comment by eddumelendez** [comment]
@chethanuk-plutoflume sorry for the delay. Can you resolve the conflicts, please?

**Comment by chethanuk-plutoflume** [comment]
> @chethanuk-plutoflume sorry for the delay. Can you resolve the conflicts, please?

Updated

**Review by eddumelendez — APPROVED** [review]

**Pull request merged**